### PR TITLE
Add Astrolabe Destination Docs

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Added documentation for new Astrolabe.so Destination.
- [netlify-build]

### Merge timing

- ASAP once approved.

### Related issues (optional)

#5156

### CODE REVIEW

File: src/connections/destinations/catalog/astrolabe/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+[Astrolabe](https://astrolabe.so/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a Revenue Operations Platform built for go-to-market teams to generate pipelines, prioritize, close, and grow accounts. It's a no-code AI-based platform that empowers teams to easily access data, build advanced predictive machine learning models, create efficient workflows, and drive better results without technical expertise.
+
+This destination is maintained by Astrolabe. For any issues with the destination, [contact the Astrolabe Support team](mailto:team@astrolabe.so).
+
+## Getting Started
+
+### Obtain your Astrolabe API Key
+
+1. Login to your [Astrolabe workspace](https://console.astrolabe.so/login){:target="_blank"}.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/astrolabe/index.md:14]:
```suggestion
1. Log in to your [Astrolabe workspace](https://console.astrolabe.so/login){:target="_blank"}.
```

File: src/connections/destinations/catalog/astrolabe/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+title: Astrolabe Destination
+id: 64d2643196f4937712e54198
+---
+
+[Astrolabe](https://astrolabe.so/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a Revenue Operations Platform built for go-to-market teams to generate pipelines, prioritize, close, and grow accounts. It's a no-code AI-based platform that empowers teams to easily access data, build advanced predictive machine learning models, create efficient workflows, and drive better results without technical expertise.
+
+This destination is maintained by Astrolabe. For any issues with the destination, [contact the Astrolabe Support team](mailto:team@astrolabe.so).
+
+## Getting Started
```
Comment (stayseesong) [inline src/connections/destinations/catalog/astrolabe/index.md:10]:
```suggestion
## Getting started
```

**Review by stayseesong — APPROVED** [review]

File: src/connections/destinations/catalog/astrolabe/index.md (hunk 2 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/astrolabe/index.md:10]: @stayseesong seems like there is a mix of "Getting Started" vs. "Getting started" throughout the catalog, should we get rid of capital "S" moving forward

Comment (pwseg) [reply src/connections/destinations/catalog/astrolabe/index.md:10]: @tcgilbert Good catch, I'll take care of it.

**Pull request merged**